mediawiker.py

```diff
@@ -345,9 +345,10 @@ def post_page(self, summary):
 
         self.view.set_scratch(True)
         utils.props.set_view_setting(self.view, 'is_changed', False)  # reset is_changed flag
-        utils.status_message('Page [[{}]] was successfully published to wiki "{}".'.format(
+        utils.status_message('Page [[{}]] was successfully published to wiki "{}"{}'.format(
```

Comment:
Maybe simplify here? Something like:

```python
msg = f'Page [[{title}]] was successfully published to wiki "{view_site}"'
if summary:
    msg = f'{msg}: "{summary}"'

print(f'{msg}.')

```

Reply:
There's no fstrings because that wasn't added until 3.6, and I guess it's not worth dropping ST3 compatibility just for that, but I moved the logic with a regular string.format